packages/api/src/constants.js

```diff
@@ -1,13 +1,17 @@
+const TenderdashRPC = require('./tenderdashRpc')
+
+let GenesisTime
+
 module.exports = {
+  EPOCH_CHANGE_TIME: Number(process.env.EPOCH_CHANGE_TIME),
   BLOCK_TIME: 5000,
-  StateTransitionEnum: {
-    DATA_CONTRACT_CREATE: 0,
-    DOCUMENTS_BATCH: 1,
-    IDENTITY_CREATE: 2,
-    IDENTITY_TOP_UP: 3,
-    DATA_CONTRACT_UPDATE: 4,
-    IDENTITY_UPDATE: 5,
-    IDENTITY_CREDIT_WITHDRAWAL: 6,
-    IDENTITY_CREDIT_TRANSFER: 7
+  get genesisTime () {
+    if (!GenesisTime) {
+      return TenderdashRPC.getGenesis().then((v) => {
+        GenesisTime = v.genesis_time
+        return v.genesis_time
+      }).catch(Error)
```

Comment:
Use named error from errors folder, or use a string `throw new Error("Could not load genesis time")`